bigquery/google/cloud/bigquery/client.py

```diff
@@ -1440,6 +1440,97 @@ def query(
 
         return query_job
 
+    def _convert_to_safe_json_dict(self, field, row_value):
+        """Maps a field and value to a JSON-safe value.
+
+        Args:
+            field ( \
+                :class:`~google.cloud.bigquery.schema.SchemaField`, \
+            ):
+                The SchemaField to use for type conversion and field name.
+
+            row_value (Union[ \
+                Sequence[list], \
+                any, \
+            ]):
+                Row value(s) to be converted. If the value is
+                a list, converted all elements in the list and return
+                a dictionary whose value is that converted list.
+
+        Returns:
+            dict:
+                A dict of a field name to value(s), encoded for JSON.
+        """
+        if not isinstance(row_value, list):
+            converter = _SCALAR_VALUE_TO_JSON_ROW.get(field.field_type)
+            value = {}
+            if converter is not None:  # STRING doesn't need converting
+                value[field.name] = converter(row_value)
+            else:
+                value[field.name] = row_value
+            return value
+
+        values = []
+        for v in row_value:
+            converter = _SCALAR_VALUE_TO_JSON_ROW.get(field.field_type)
+            if converter is not None:  # STRING doesn't need converting
+                values.append(converter(v))
+            else:
+                values.append(v)
+        return {field.name: values}
+
+    def _handle_nested_schema(self, field, row_value):
```

Comment:
I'd prefer this be named `_field_value_to_json`, to match the naming convention in `_helpers.py`.